pkg/deployments/tasks/20210418-vault/abi/BalancerHelpers.json

```diff
@@ -68,7 +68,7 @@
         "type": "uint256[]"
       }
     ],
-    "stateMutability": "view",
+    "stateMutability": "nonpayable",
```

Comment:
Ah unfortunate. We have the `overrideQueryFunctions` helper change this, but not on the build info itself.